valhalla/baldr/graphreader.h

```diff
@@ -18,6 +18,9 @@
 #include <valhalla/midgard/aabb2.h>
 #include <valhalla/midgard/pointll.h>
 #include <valhalla/midgard/sequence.h>
+#if defined(_MSC_VER)
+#define stat _stat64
+#endif
```

Comment:
we get stat from sequence.h. would you mind moving this over there?

Reply:
oh i see its there already, this is superfluous then.

Reply:
yeah, I think I first patched it in graphreader, then realized it was in sequence as well, so it got put both places...